### ISSUE TITLE
Examples for saving htmlwidgets and plots

### ISSUE THREAD

**Issue by nebuzoltanszuhai** [datastorm-open/rAmCharts#68]
I was playing a bit with the different charts available under rAmCharts (that looks impressive btw), and now wondering, if you plan to add functions or examples especially for htmlwidgets save, that I could not make working with this package. With leaflet, plotly, etc it is really easy. 

Thanks, Zoltan

**Comment by bthieurmel** [comment]
Hi,

You have to save the ``plot`` object. A ``am`` object become a ``htmlwidgets`` after plot.

````
am <- amBoxplot(rnorm(100))
am
htmlwidgets::saveWidget(plot(am), file = "am.html")
````

We are adding a wrapper save function for next version

**Comment by nebuzoltanszuhai** [comment]
Thanks for the quick reply!
Okay, then we are closer, I tried exactly the same with amBarplot and amAngularGauge where the same method did not work, it looks like it does work for the amBoxPlot without issue.

p <- amAngularGauge(...)
htmlwidgets::saveWidget(p, file="test_widget.html")

of course parameters, bands are filled.

What error message I got:
"Error in x$width : $ operator not defined for this S4 class"

**Comment by bthieurmel** [comment]
Yes, so you 've just have to add plot() saving widget, waiting for a simpler function.

````
p <- amAngularGauge(x = 10)
htmlwidgets::saveWidget(plot(p), file="test_widget.html")
````

**Comment by nebuzoltanszuhai** [comment]
Flawless, thanks! (I was sure I tried this as well, but apparently looks like not.)
